config/envs/development.py

```diff
@@ -57,7 +57,7 @@ class DevelopmentConfig(Config):
     USE_LOCAL_DATA = strtobool(getenv("USE_LOCAL_DATA", "True"))
     SESSION_COOKIE_SECURE = False
     AUTO_REDIRECT_LOGIN = True
-    DISABLE_NOTIFICATION_SERVICE = False  # Toggle on if you have no notify api key.
+    DISABLE_NOTIFICATION_SERVICE = True  # Toggle on if you have no notify api key.
```

Comment:
I'm not sure that this change should be committed.

If we do want to change it, maybe we should make it at least configurable eg `bool(strtobool(getenv("DISABLE_NOTIFICATION_SERVICE", "false")))`